Makefile

```diff
@@ -70,5 +68,13 @@ docs: init install
 server:
 	examples/vasp/server.sh -p 8080
 
+docker:
+	docker-compose -f docker/testnet/docker-compose.yaml up --detach
+
+docker-down:
+	docker-compose -f docker/testnet/docker-compose.yaml down -v
+
+docker-stop:
+	docker-compose -f docker/testnet/docker-compose.yaml stop
 
-.PHONY: init check lint format install test cover build diemtypes protobuf gen dist pylama docs server
+.PHONY: init lint format test cover build diemtypes protobuf gen dist docs server docker docker-down docker-stop
```

Comment:
is this a set of operations to be run in order?

Reply:
no, just mark these targets as phony targets.